.env.example

```
@@ -0,0 +1,65 @@
+# ── Local Development Setup (bin/dc_prep) ────────────────────
+# These values are used by the unified setup script.
+# Override on the command line:
+#   bash bin/dc_prep --tenant safetrust --endpoint http://localhost:8080 --secret myadminsecretkey
+SETUP_TENANT=safetrust
+SETUP_HASURA_ENDPOINT=http://localhost:8080
+SETUP_ADMIN_SECRET=myadminsecretkey
+
+# ── PostgreSQL ───────────────────────────────────────────────
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgrespassword
+POSTGRES_DB=safetrust
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_DB key should go before the POSTGRES_PASSWORD key

(UnorderedKey)